scripts/mysql_system_tables_fix.sql

```diff
@@ -1303,7 +1303,9 @@ DROP PREPARE stmt;
 #  GRANT ALL PRIVILEGES ON *.* TO 'mysql.pxc.internal.session'@localhost WITH GRANT OPTION;
 #  GRANT BACKUP_ADMIN, LOCK TABLES, PROCESS, RELOAD, REPLICATION CLIENT, SUPER ON *.* TO 'mysql.pxc.internal.session'@localhost WITH GRANT OPTION;
 #
-INSERT IGNORE INTO mysql.user VALUES ('localhost','mysql.pxc.internal.session','Y','Y','Y','Y','Y','Y','Y','Y','Y','Y','Y','Y','Y','Y','Y','Y','Y','Y','Y','Y','Y','Y','Y','Y','Y','Y','Y','Y','Y','','','','',0,0,0,0,'caching_sha2_password','$A$005$THISISACOMBINATIONOFINVALIDSALTANDPASSWORDTHATMUSTNEVERBRBEUSED','N',CURRENT_TIMESTAMP,NULL,'Y','Y','Y',NULL,NULL,NULL,NULL);
+INSERT IGNORE INTO mysql.user
+(host, user, select_priv, plugin, authentication_string, ssl_cipher, x509_issuer, x509_subject)
+VALUES ('localhost','mysql.pxc.internal.session','Y','caching_sha2_password','$A$005$THISISACOMBINATIONOFINVALIDSALTANDPASSWORDTHATMUSTNEVERBRBEUSED','','','');
```

Comment:
why this INSERT statement is changed? just simplified?

Reply:
I had made these fixes few months back and it had fixed a few test failures. But now, I have forgotten the exact reason. I'll dig into it and will let you know.

Reply:
I can't recall the reason for this change. When I tried with reverting this change, it didn't cause any test failures in my local environment. So, I have submitted to Jenkins with reverted changes for further testing.

Branch: https://github.com/venkatesh-prasad-v/percona-xtradb-cluster/tree/8.0-PXC-3641-jenkins
Full suite: https://pxc.cd.percona.com/view/PXC%208.0/job/pxc-8.0-param/289/console
Galera suite: https://pxc.cd.percona.com/view/PXC%208.0/job/pxc-8.0-param/290/console

Reply:
ok, lets revert it

Reply:
I'll wait for a clean Jenkins and will then update the PR.

Reply:
:+1:

Reply:
Test results were good after reverting the INSERT statement changes, I have updated the PR with the required changes.